www/docs/style/sass/_twfy-mixins.scss

```diff
@@ -307,7 +307,6 @@ input.form-control {
     font-size: inherit;
     line-height: inherit;
     padding: 0.35em 0.4em;
-    margin: .2em 0 0 0;
```

Comment:
Could we not remove it but go back to `margin: 0;` as it was before, which still fixes this but doesn't add extra margin bottom to places we might not expect (not that it really matters I suspect).

Reply:
Can do!